src/openforms/forms/tests/admin/test_views.py

```diff
@@ -305,3 +308,271 @@ def test_single_import(self, m_import):
             "Form successfully imported!",
             messages[0].message,
         )
+
+
+@disable_admin_mfa()
+class TestMigrationForm(WebTest):
+    csrf_checks = False
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.admin_user = StaffUserFactory(user_permissions=["forms.change_form"])
+
+    def test_not_superuser_cant_access(self):
+        user = UserFactory(is_staff=False, is_superuser=False)
+        ogone_merchant = OgoneMerchantFactory()
+        form_to_migrate = FormFactory(
+            name="Ogone form 1",
+            payment_backend="ogone-legacy",
+            payment_backend_options={"merchant_id": ogone_merchant.pk},
+        )
+
+        redirect = self.app.post(
+            reverse("admin:forms_payment_migration"),
+            {"forms_to_migrate": [form_to_migrate.pk]},
+            user=user,
+        )
+
+        redirect_url = (
+            f"{reverse('admin:login')}?next={reverse('admin:forms_payment_migration')}"
+        )
+        self.assertRedirects(
+            redirect,
+            redirect_url,
+            fetch_redirect_response=False,
+            target_status_code=302,
+        )
+
+        form_to_migrate.refresh_from_db()
+
+        self.assertEqual(form_to_migrate.payment_backend, "ogone-legacy")
+        self.assertEqual(
+            form_to_migrate.payment_backend_options, {"merchant_id": ogone_merchant.pk}
+        )
+
+    def test_form_permissions(self):
```

Comment:
please take care in naming your tests appropriately and be verbose.

This test name only tells me something with permissions is being tested, but it's not clear if permissions should allow or block the operation.


```suggestion
    def test_staff_user_without_change_form_permissions_has_no_access(self):
```